RKNPS/RKNPS-IOC-01App/Db/DFKPS_RIKEN.db

```diff
@@ -21,7 +21,7 @@ record(bo, "$(P)POWER:SP")
     field(SIML, "$(P)SIM")
     field(SIOL, "$(P)SIM:POWER:SP PP")
     field(SDIS, "$(P)DISABLE")
-    info(autosaveFields, "VAL")
+    field(TPRO, "1")
```

Comment:
Note for reviewer: I have left TPRO on intentionally as it is useful for diagnostics and shouldn't log a huge amount.